runtime/src/main/java/org/corfudb/protocols/wireprotocol/LogData.java

```diff
@@ -81,6 +82,8 @@ public Object getPayload(CorfuRuntime runtime) {
                             } catch (Throwable throwable) {
                                 log.error("Exception caught at address {}, {}, {}",
                                         getGlobalAddress(), getStreams(), getType());
+                                log.error("Raw data buffer {}",
+                                        copyBuf.resetReaderIndex().toString(Charset.defaultCharset()));
```

Comment:
It seems like this should really be in the serializer's implementation and not here. Would this be useful if the payload is not a json object? Not sure how helpful it is to dump binary pay loads to a log file.

Reply:
Also, there might be an issue if the payload has sensitive information and it shouldn't be logged to a logging file.

Reply:
If it is not json at least the byte buffer when dumped can be interpreted somehow for debugging purposes to decipher what the error is.
As for the sensitive info, I am implementing a specific ask for a scenario that should never happen in production.